From 43800886ed311e792045f103baa64e780c9f9c60 Mon Sep 17 00:00:00 2001
From: Pavel Hrdina <phrdina@redhat.com>
Date: Tue, 26 Jul 2016 09:32:14 +0200
Subject: [virt-manager PATCH 1/3] translation: mark some strings to be
 translated

Signed-off-by: Pavel Hrdina <phrdina@redhat.com>
(cherry picked from commit 810c73864a5af4ff6e4b725603fc20771137dba2)

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1271152

Signed-off-by: Pavel Hrdina <phrdina@redhat.com>
---
 virtManager/create.py      | 4 ++--
 virtManager/storagelist.py | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/virtManager/create.py b/virtManager/create.py
index 737c1fa..7d53110 100644
--- a/virtManager/create.py
+++ b/virtManager/create.py
@@ -76,11 +76,11 @@ def _pretty_arch(_a):
 
 
 def _pretty_storage(size):
-    return "%.1f GiB" % float(size)
+    return _("%.1f GiB") % float(size)
 
 
 def _pretty_memory(mem):
-    return "%d MiB" % (mem / 1024.0)
+    return _("%d MiB") % (mem / 1024.0)
 
 
 ###########################################################
diff --git a/virtManager/storagelist.py b/virtManager/storagelist.py
index c16cf5c..4c93c64 100644
--- a/virtManager/storagelist.py
+++ b/virtManager/storagelist.py
@@ -336,7 +336,7 @@ class vmmStorageList(vmmGObjectUI):
         self.widget("pool-name-entry").set_text(pool.get_name())
         self.widget("pool-name-entry").set_editable(not active)
         self.widget("pool-sizes").set_markup(
-                """%s Free / <i>%s In Use</i>""" %
+                _("%s Free / <i>%s In Use</i>") %
                 (pool.get_pretty_available(), pool.get_pretty_allocation()))
         self.widget("pool-location").set_text(
                 pool.get_target_path())
-- 
2.10.0

